vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -195,6 +195,7 @@ type internal FSharpCheckerWorkspaceServiceFactory
 
 type
     [<Guid(FSharpCommonConstants.packageGuidString)>]
+    [<ProvideLanguageEditorOptionPage(typeof<Options.IntelliSenseOptionPage>, "F#", null, "IntelliSense", "6008")>]
```

Comment:
@cloudRoutine search the solution for such numbers. They sit somewhere in a res file, I've added one when implementing the initial intellisense prop page.